public/js/attendance-scripts/attendance-utilities.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * PEP Capping 2017 Algozzine's Class
+ *
+ * javascript utility functions
+ *
+ * functions used to reduce duplicated code
+ *
+ * @author Scott Hansen
+ * @copyright 2017 Marist College
+ * @version [version number]
+ * @since [initial version number]
+ */
+
+function populateTimes() {
+    var selectTime = document.getElementById('time-input');
+
+    //populate am first, then pm
+    for(var z = 0; z <= 1; z++) {
+        var amORpm = (z === 0) ? "AM" : "PM";
+
+        //AM
```

Comment:
Remove `//AM` and rename `var hourAM` to just `hour` since you're covering both cases in this loop now.